### PR TITLE
Funcs v0.4.7

### PR DESCRIPTION
**Pull request by yjb-ds**

### CODE REVIEW

File: examples/Untitled.ipynb [hunk 1]
```diff
@@ -0,0 +1,228 @@
+{
```
Comment (yaronha) [inline examples/Untitled.ipynb:1]: @yjb-ds do we need that file?

File: mlrun/mlutils/plots.py [hunk 2]
```diff
@@ -75,7 +75,7 @@ def plot_roc(
         plt.plot(fpr, tpr, label=f"positive class")
 
     fname = f"{plots_dir}/{key}.html"
-    return context.log_artifact(PlotArtifact(key, body=plt.gcf()), local_path=fname)
+    context.log_artifact(PlotArtifact(key, body=plt.gcf()), local_path=fname)
```
Comment (yaronha) [inline mlrun/mlutils/plots.py:78]: @yjb-ds should keep the `return`

File: mlrun/mlutils/plots.py [hunk 3]
```diff
@@ -114,4 +114,4 @@ def plot_importance(
 
     # feature importances are also saved as a csv table (generally small):
     fname = key + "-tbl.csv"
-    return context.log_artifact(TableArtifact(key + "-tbl", df=feature_imp), local_path=fname)
+    context.log_artifact(TableArtifact(key + "-tbl", df=feature_imp), local_path=fname)
```
Comment (yaronha) [inline mlrun/mlutils/plots.py:117]: @yjb-ds should keep the `return`

File: examples/Untitled.ipynb (hunk 1 above)
Comment (yjb-ds) [reply examples/Untitled.ipynb]: deleted

File: mlrun/mlutils/plots.py (hunk 3 above)
Comment (yjb-ds) [reply mlrun/mlutils/plots.py]: done

**Pull request closed without merging**